README.md

```diff
@@ -1,45 +1,39 @@
-# General\_METAB\_ANALYSIS PIPELINE
+# 16S pipeline from the Sunagawa Lab
+### Important updates
+- "blacklist" is now called "blocklist"
+- The new added UPARSE function can take a few hours to run depending on data size. Credit to [@cmfield](https://github.com/cmfield) for the code.
+- We added new options to the config file: 
+  - Forward and reverse primer sequences, in case you don't have a primers file
+  - Reference sequence file for the usearch rule
```

Comment:
Don't confuse the 'reference sequence file' with the silva database used in 8uparsetax